### ISSUE TITLE
Dark mode and transparent images

### ISSUE THREAD

**Issue by hujambo-dunia** [galaxyproject/training-material#6056]
Dark mode as a default needs a rethink because of transparent images which will fail accessibility because they are totally illegible. Below example is from the [Deep Learning (Part 2) - Recurrent neural networks (RNN)](https://training.galaxyproject.org/training-material/topics/statistics/tutorials/RNN/tutorial.html) tutorial.

<img width="636" alt="Image" src="https://github.com/user-attachments/assets/f000b7b3-6118-4f43-b2f4-dcd562172509" />

**Comment by shiltemann** [comment]
@hujambo-dunia Thanks for reporting this, I think @scorreard addressed this in https://github.com/galaxyproject/training-material/pull/6184, so I will close this now, but feel free to re-open if you notice it still happening
